### ISSUE TITLE
Land in-line initial condition interpolation fails with tri-grid (ne30_r05_oECv3)

### ISSUE THREAD

**Issue by golaz** [E3SM-Project/E3SM#3276]
I'm trying out the new tri-grid functionality (48e617270e311a53a6f6bb26e05e9011893aeaac) with compset **F20TRC5-CMIP6** and resolution  **ne30_r05_oECv3**.

The model runs with land cold start; but when I try to use in-line interpolation for land initial conditions (finidat_interp), the model fails during land initialization:

```
100: e3sm.exe           00000000029D9E3D  pio_spmd_utils_mp         294  pio_spmd_utils.F90.in
100: e3sm.exe           00000000029C0048  box_rearrange_mp_         302  box_rearrange.F90.in
100: e3sm.exe           00000000029645F9  rearrange_mp_rear          80  rearrange.F90.in
100: e3sm.exe           000000000297E127  piodarray_mp_writ         583  piodarray.F90.in
100: e3sm.exe           0000000002982B00  piodarray_mp_writ         221  piodarray.F90.in
100: e3sm.exe           000000000241BFB5  ncdio_pio_mp_ncd_        1595  ncdio_pio.F90.in
100: e3sm.exe           0000000001B99812  initinterpmod_mp_         993  initInterp.F90
100: e3sm.exe           0000000001ABDAB8  clm_initializemod         771  clm_initializeMod.F90
100: e3sm.exe           0000000001AA8118  lnd_comp_mct_mp_l         281  lnd_comp_mct.F90
100: e3sm.exe           000000000043C5D4  component_mod_mp_         257  component_mod.F90
100: e3sm.exe           000000000042ACC4  cime_comp_mod_mp_        1343  cime_comp_mod.F90
100: e3sm.exe           0000000000439519  MAIN__                    122  cime_driver.F90
```

The last line in the land log file is:

` Interpolating: FSD24_VALUE`

so the model might be failing when interpolating `FSD24_VALUE`

For in-line interpolation, my settings were

```
finidat = ' '
finidat_interp_source = '/compyfs/zhan524/restart/20180316.DECKv1b_A1.ne30_oEC.edison.1980-01-01/20180316.DECKv1b_A1.ne30_oEC.edison.clm2.r.1980-01-01-00000_DECKv1.nc' 
finidat_interp_dest = 'finidat_interp_20180316.DECKv1b_A1.ne30_oEC.edison.1980-01-01.clm2.r.1980-01-01-00000.nc'
```
Hoping that @bishtgautam can help with this.

**Comment by bishtgautam** [comment]
i will look into this today

**Comment by bishtgautam** [comment]
The fix for inline interpolation (#3214) was added to added to master after tri-grid functionality was added to master. Could you test the latest master?

**Comment by golaz** [comment]
@bishtgautam : I was using daa9299f12, which includes the fix for in-line interpolation.

**Comment by bishtgautam** [comment]
I successfully ran a `F20TRC5-CMIP6.ne30_r05_oECv3` with [interpolation settings](https://github.com/E3SM-Project/E3SM/issues/3276#issue-515824834) provided by @golaz using the fix issued in PR-#3286.

**Comment by golaz** [comment]
Verified that it is fixed in https://github.com/E3SM-Project/E3SM/pull/3286#pullrequestreview-311271620
